### DISCUSSION TITLE
Package Installer for Debian 12?

### DISCUSSION THREAD

**Discussion by imager2019** [inventree/InvenTree#5129] [Q&A]
Wondering what the future looks like for an update to the installer for use with Debian 12?

The current installer reports that Debian GNU/Linux 12 is not supported. 

Might there be an update to the installer coming soon for later OS versions??

John

**Comment by matmair** [comment]
Bookworm hasn't been out a month, I do not have enough runtime on it to consider it a stable deploy target.

**Reply by imager2019** [reply]
Good to know - I have gone ahead and setup Debian 11 and have the install setup. Thanks for the update.
